leaf-ui/js/analysis.js

```diff
@@ -15,12 +15,12 @@ var analysisResult;
 var savedAnalysisData;
 var graph;
 var current;
-// Analysis Configuration map (key: configId, value: analysisConfig object)
-var analysisMap = new Map();
-// Global variable to keep track of what analysis configuration is currently being used
-var currAnalysisConfig;
-// Count for number of analysis configurations
-var configCount = 0;
+// // Analysis Configuration map (key: configId, value: analysisConfig object)
```

Comment:
Since you directly copy pasted this into displayAnalysis, it might be better for now to delete these commented out lines and instead leave a TODO note saying that the analysis configuration variables moved to displayAnalysis, since we need to come back to this to figure out the importance of the other vars that live here and how they integrate into what we're doing

Reply:
Done